.github/workflows/test.yml

```diff
@@ -213,7 +219,7 @@ jobs:
 
       - name: Test
         run: |
-          wget -O - https://raw.githubusercontent.com/KomodoPlatform/komodo/v0.8.1/zcutil/fetch-params-alt.sh | bash
+          wget -O - https://raw.githubusercontent.com/KomodoPlatform/komodo/0adeeabdd484ef40539d1275c6a765f5c530ea79/zcutil/fetch-params-alt.sh | bash
```

Comment:
We need version `v.0.8.1` here to be compatible with the utxo docker nodes we use ref. https://github.com/KomodoPlatform/komodo-defi-framework/pull/2191/commits/9b94bf059c9785b0e2abcadf9ea65ede30600d78

Reply:
We have some failing zcoin tests https://github.com/KomodoPlatform/komodo-defi-framework/actions/runs/11009838532/job/30709050628?pr=2217#step:7:1378 probably due to using old zcash params, we only need `v.0.8.1` for docker tests.

Reply:
> We shouldn't rely on multiple versions of the same script for different parts of the project :/

It's used by the UTXO node, we need to update it to use the same version everywhere. Will open an issue.

Reply:
This one https://hub.docker.com/r/artempikulin/testblockchain/tags

Reply:
Issue: https://github.com/KomodoPlatform/komodo-defi-framework/issues/2231

Reply:
mmm, same tests are still failing with `ZCashParamsNotFound`. This doesn't happen in dev.

Reply:
Problem with wget https://github.com/KomodoPlatform/komodo-defi-framework/actions/runs/11054265357/job/30710639552?pr=2217#step:7:20

Reply:
https://github.com/KomodoPlatform/komodo-defi-framework/pull/2217/commits/c8bd51cba71fa56c55185914d9aae8784b032e3b should fix that.